File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.py
def solution(s):
    length = len(s)
    mid = length // 2

    if length % 2 == 0:
        return s[mid - 1:mid + 1]  # 짝수: 가운데 두 글자
    else:
        return s[mid]  # 홀수: 가운데 한 글자

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    const len = s.length;
    const mid = Math.floor(len / 2);
    
    return len % 2 === 0 
        ? s.slice(mid - 1, mid + 1)  // 짝수: 가운데 두 글자
        : s.charAt(mid);            // 홀수: 가운데 한 글자
}
